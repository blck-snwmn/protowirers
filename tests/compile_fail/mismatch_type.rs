use protowirers::*;

#[derive(Proto)]
struct Sample {
    #[def(field_num = 1, def_type = "sint64")]
    s: u32,
}

fn main() {}
