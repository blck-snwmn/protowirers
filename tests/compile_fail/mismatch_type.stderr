error: defined def_type `Sint64` does not match this Rust type
 --> $DIR/mismatch_type.rs:6:8
  |
6 |     s: u32,
  |        ^^^
